src/components/DynamicFormik/DynamicFormik.js

```diff
@@ -9,6 +9,7 @@ import { settings } from 'carbon-components';
 import {
```

Comment:
I'm not seeing in this file now where its being added to the DynamicFormik validation starting on line 80 in the function `generateYupAst`. It should be added in here and we should look at the different validations we can do that make sense for a date input field